### ISSUE TITLE
Static method calls are missing class name

### ISSUE THREAD

**Issue by MikeHillberg** [dotnet/docs#18561]
In several places the sample code is missing class names in static method calls. For example it's `Console.WriteLine(...)`, not `WriteLine(...)`. And it's `SyntaxFactory.IdentifierName("System")` not `IdentifierName("System")`.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 442d453e-38dd-ed79-741c-813429a41822
* Version Independent ID: 5f1f64a1-9d23-0010-b505-d3874041f8f2
* Content: [Get started with syntax transformation (Roslyn APIs)](https://docs.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/get-started/syntax-transformation)
* Content Source: [docs/csharp/roslyn-sdk/get-started/syntax-transformation.md](https://github.com/dotnet/docs/blob/master/docs/csharp/roslyn-sdk/get-started/syntax-transformation.md)
* Product: **dotnet-csharp**
* Technology: **csharp-roslyn**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by BillWagner** [comment]
Thanks for pointing this out @MikeHillberg 

We were experimenting with `static using` in several samples. We've learned that doesn't work as well in our sample code as it does in shared projects.

The `static using` statements should be removed, and the class names should be added back on the static calls.

I've added this to our backlog, and added the "up for grabs" label if community members want to submit a pull request before we update this.
